designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -36,9 +36,9 @@ import scala.language.higherKinds
 object ProcessService {
   type EmptyResponse = XError[Unit]
 
-  @JsonCodec case class CreateProcessCommand(processName: ProcessName, category: String, isSubprocess: Boolean)
+  @JsonCodec case class CreateProcessCommand(processName: ProcessName, category: String, isSubprocess: Boolean, forwardedUserName: Option[String] = None)
 
-  @JsonCodec case class UpdateProcessCommand(process: DisplayableProcess, comment: UpdateProcessComment)
+  @JsonCodec case class UpdateProcessCommand(process: DisplayableProcess, comment: UpdateProcessComment, forwardedUserName: Option[String] = None)
```

Comment:
So you are saying that any user will be able to send `UpdateProcessCommand` with any other `forwardedUserName`? :D

Reply:
I've added new Permission to protect from overriding user name